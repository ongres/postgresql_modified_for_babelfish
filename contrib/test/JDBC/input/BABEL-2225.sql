CREATE SEQUENCE seq_2225 INCREMENT BY 6 MINVALUE 5 MAXVALUE 10;
GO

CREATE SEQUENCE seq_2225 INCREMENT BY 6 MINVALUE 5 MAXVALUE 11;
GO

ALTER SEQUENCE seq_2225 MAXVALUE 10;
GO

ALTER SEQUENCE seq_2225 INCREMENT BY 7;
GO

DROP SEQUENCE seq_2225;
GO

-- Test with extreme values
CREATE SEQUENCE seq_2225 INCREMENT BY -9223372036854775808 MINVALUE 0 MAXVALUE 9223372036854775807;
GO

CREATE SEQUENCE seq_2225 INCREMENT BY -9223372036854775808 MINVALUE -9223372036854775808 MAXVALUE 9223372036854775807;
GO

DROP SEQUENCE seq_2225;
GO
