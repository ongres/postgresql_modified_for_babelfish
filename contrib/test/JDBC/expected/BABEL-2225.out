CREATE SEQUENCE seq_2225 INCREMENT BY 6 MINVALUE 5 MAXVALUE 10;
GO
~~ERROR (Code: 11701)~~

~~ERROR (Message: The absolute value of the increment must be less than or equal to the difference between the minimum and maximum value of the sequence object.)~~


CREATE SEQUENCE seq_2225 INCREMENT BY 6 MINVALUE 5 MAXVALUE 11;
GO

ALTER SEQUENCE seq_2225 MAXVALUE 10;
GO
~~ERROR (Code: 11701)~~

~~ERROR (Message: The absolute value of the increment must be less than or equal to the difference between the minimum and maximum value of the sequence object.)~~


ALTER SEQUENCE seq_2225 INCREMENT BY 7;
GO
~~ERROR (Code: 11701)~~

~~ERROR (Message: The absolute value of the increment must be less than or equal to the difference between the minimum and maximum value of the sequence object.)~~


DROP SEQUENCE seq_2225;
GO

-- Test with extreme values
CREATE SEQUENCE seq_2225 INCREMENT BY -9223372036854775808 MINVALUE 0 MAXVALUE 9223372036854775807;
GO
~~ERROR (Code: 11701)~~

~~ERROR (Message: The absolute value of the increment must be less than or equal to the difference between the minimum and maximum value of the sequence object.)~~


CREATE SEQUENCE seq_2225 INCREMENT BY -9223372036854775808 MINVALUE -9223372036854775808 MAXVALUE 9223372036854775807;
GO

DROP SEQUENCE seq_2225;
GO
